pkg/graphql/schema.go

```diff
@@ -72,6 +72,9 @@ const schema string = `
 
         # IPLD block data for the Receipt this Log exists in.
         ipldBlock: Bytes!
+
+		# Status of the Receipt IPLD block this Log exists in.
```

Comment:
Nit: Indentation is different from above.

Reply:
Done